### PR TITLE
Simplified getArrayCopy method

### LINKED ISSUE #52: LineDataSet: 'fill' not working
Hi,

The fill attribute of LineDataSet seems to have no effect on the generated code. This is due to its PHPDoc var type being `bool|string` and the type comparison in [ArraySerializable](https://github.com/halfpastfouram/PHPChartJS/blob/master/src/Delegate/ArraySerializable.php#L39) only working for `bool` or `boolean`

Coming-up with a PR soon.

### PR DESCRIPTION
**Pull request by artggd**
Simplified the property access logic in `getArrayCopy` method

This fixes #52 

I also added a unit test.

### CODE REVIEW

**Comment by artggd** [comment]
(as a side note, this solution works (for `is` and `get`) but I would recommend using a property access library such as [Symfony's PropertyAccess Component](http://symfony.com/doc/current/components/property_access.html))

**Comment by halfpastfouram** [comment]
Excuse me for the late reply. I don't want you to get the idea I'm not looking into this. I've yet to find some time to actually compare and check your changes as I am very busy at work, currently. Don't worry, I'll get to it in a few days.

**Comment by artggd** [comment]
All good. There's no urge 😊

File: src/Delegate/ArraySerializable.php [hunk 1]
```diff
@@ -34,6 +34,10 @@ public function getArrayCopy()
 					// If 'getSomething' doesn't exist, try to use 'isSomething'
 					$getter = 'is' . ucfirst( $property );
 				}
+				// Finally simply use the property
+				if( !method_exists($this, $getter) ) {
+					$getter = $property;
+				}
```
Comment (halfpastfouram) [inline src/Delegate/ArraySerializable.php]: I see the tests are running fine when I comment out this addition, so I will update the tests to try and make sure it fails in the case you described in #52.

**Pull request merged**